src/mmw/js/src/analyze/models.js

```diff
@@ -54,6 +54,33 @@ var AnalyzeTaskModel = coreModels.TaskModel.extend({
     }
 });
 
+var AnalyzeTaskCollection = Backbone.Collection.extend({
+    model: AnalyzeTaskModel
+});
+
+AnalyzeTaskCollection.getSingleton = function(App, aoi) {
```

Comment:
This is just stylistic, but couldn't you include this and the next function in the object passed to `Collection.extend` above?